src/steps/Background/CurrentContract/CurrentContract.vue

```diff
@@ -16,51 +16,90 @@
               id="currentContractOptions"
               :card="true"
               :items="currentContractOptions" 
-              :value.sync="currentContractOption"
+              :value.sync="currentContractExists"
             />
           </div>
-
         </v-col>
       </v-row>
     </v-container>
 </template>
+
 <script lang="ts">
-import Vue from "vue";
-import { Component } from "vue-property-decorator";
+import { Component, Mixins } from "vue-property-decorator";
+
 import ATATRadioGroup from "@/components/ATATRadioGroup.vue"
+
 import { RadioButton } from "../../../../types/Global";
-import Background from "@/store/background";
+import AcquisitionPackage from "@/store/acquisitionPackage";
+import SaveOnLeave from "@/mixins/saveOnLeave";
+import { CurrentContractDTO } from "@/models/BackgroundDTOs";
+import { hasChanges } from "@/helpers";
 
 @Component({
   components: {
     ATATRadioGroup,
   },
 })
-export default class CurrentContract extends Vue {
+
+export default class CurrentContract extends Mixins(SaveOnLeave) {
   private currentContractOptions: RadioButton[] = [
     {
       id: "Yes",
       label: "Yes. There is a current contract for this effort.",
-      value: "Yes",
+      value: "true",
     },
     {
       id: "No",
       label: "No. This is a new requirement.",
-      value: "No",
+      value: "false",
     },
   ];
 
-  public get currentContractOption(): string {
-    const hasCurrentContract = Background.hasCurrentContract;
-    if (hasCurrentContract !== null) {
-      return hasCurrentContract ? "Yes" : "No";
+  public currentContractExists 
+    = AcquisitionPackage.currentContract?.current_contract_exists || "";
+
+  private get currentData(): CurrentContractDTO {
+    return {
+      current_contract_exists: this.currentContractExists,
+    };
+  }
+
+  private savedData: CurrentContractDTO = { 
+    current_contract_exists: "" 
```

Comment:
By setting the saved data  current_contract_exists to an empty string will that will make the save happen everytime unless the current data is empty as well.   Am I missing something?